servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -732,6 +733,7 @@ float light_process_spot_shadow(uint idx, vec3 vertex, vec3 normal) {
 				}
 
 				shadow /= float(sc_penumbra_shadow_samples);
+				shadow = shadow * spot_lights.data[idx].shadow_opacity + 1.0 - spot_lights.data[idx].shadow_opacity;
```

Comment:
use mix(), this looks pretty weird.

Reply:
I hadn't thought about using `mix()` here, it makes sense :slightly_smiling_face: